docs/build/getting-started/networks-endpoints.mdx

```diff
@@ -49,9 +49,9 @@ Mainnet.
 
 <AddToMetaMaskButton cfg={EVMNetworks['shimmerevm']} />
 
-| Base Token    | Protocol  | Chain ID | RPC URL                              | Explorer                             |
-| ------------- | --------- | -------- | ------------------------------------ | ------------------------------------ |
-| Shimmer Token | ISC / EVM | 148      | https://json-rpc.evm.shimmer.network | https://explorer.evm.shimmer.network |
+| Base Token    | Protocol  | Chain ID | RPC URL                              |Websocket RPC URL                     | Explorer                             |
+| ------------- | --------- | -------- | ------------------------------------ | ------------------------------------ | ------------------------------------ |
+| Shimmer Token | ISC / EVM | 148      | https://json-rpc.evm.shimmer.network | wss://ws.json-rpc.evm.shimmer.network/ |  https://explorer.evm.shimmer.network |
```

Comment:
Isn't the wss RPC url the same thing as a normal RPC url just over wss?
Would it make sense to add it to the same column instead of creating a new one?
```suggestion
| Base Token    | Protocol  | Chain ID | RPC URL                                                                       | Explorer                             |
| ------------- | --------- | -------- | ----------------------------------------------------------------------------- | ------------------------------------ |
| Shimmer Token | ISC / EVM | 148      | https://json-rpc.evm.shimmer.network or wss://ws.json-rpc.evm.shimmer.network | https://explorer.evm.shimmer.network |
```

Reply:
Seems we dont provide one, nope - Dave usually announces them but not for the new Testnet it seems